src/noc/guards/noc-guard.spec.ts

```diff
@@ -92,7 +92,8 @@ describe('NocGuard', () => {
         'caseworker-ia-iacjudge',
         'caseworker-publiclaw-judiciary'
       ],
-      showFindCase: false
+      showFindCase: false,
+      showCaseReferenceSearchBox: false
```

Comment:
@johnbenjamin-hmcts - what happens if you omit `showCaseReferenceSearchBox: false` from this test - does it fail? (Just wondering whether changes are required for something that isn't part of Global Search.)